elasticdl/docker/Dockerfile.dev

```diff
@@ -51,5 +51,8 @@ RUN python /var/frappe_recordio_gen.py --data /root/.keras/datasets --output_dir
 # Copy heart dataset
 COPY elasticdl/python/data/recordio_gen/heart_recordio_gen.py /var/heart_recordio_gen.py
 RUN python /var/heart_recordio_gen.py --data_dir /root/.keras/datasets --output_dir /data/heart
+# Copy heart dataset
+COPY elasticdl/python/data/recordio_gen/census_recordio_gen.py /var/census_recordio_gen.py
```

Comment:
This script will trigger the data download from https://archive.ics.uci.edu/ml/datasets/census+income.
The network is not good enough to connect to this URL, the download may fail if we add it in the regular CI.

Reply:
But other scripts also need to download datasets such as frappe dataset, heart dataset.

Reply:
Yes, other dataset source URL is stable enough. I have tried this data source URL, the error rate is not low.

Reply:
OK, we can try to search for another source URL for the dataset.